examples/positron-django/src/webapp/settings.py

```diff
@@ -2,11 +2,11 @@
 
 Generated by 'django-admin startproject' using Django 4.1.1.
 
-For more information on this file, see
-https://docs.djangoproject.com/en/4.1/topics/settings/
+For more information on this file, see:
+- https://docs.djangoproject.com/en/4.1/topics/settings/
 
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/4.1/ref/settings/
+For the full list of settings and their values, see:
+- https://docs.djangoproject.com/en/4.1/ref/settings/
```

Comment:
`docformatter` really wanted to insert a bunch of newlines here so i added some symbols to try to stop that from happening.

- https://github.com/PyCQA/docformatter/issues/180

Reply:
Interesting that it doesn't add ticks around the URL like it did in the [briefcase patch](https://github.com/beeware/briefcase/pull/1200/files#diff-36b2a8ba03657d052f5c4689aa2fd4559c7023cddd313d18b899ad422a421054R64)

Reply:
Oh....i actually did that manually in the briefcase update. I had forgotten I did that.